src/tabs.scss

```diff
@@ -41,24 +81,72 @@ $block: #{$fd-namespace}-tabs;
 
   @include fd-var-color("border-bottom-color", $fd-tabs-background-color, --fd-tabs-background-color);
 
+  &__count {
+    @include fd-reset();
+
+    line-height: 1rem;
+    padding-bottom: 0.125rem;
+    color: $fd-tabs-label-color;
+  }
+
   &__item {
     @include fd-reset();
 
-    padding: 0 $fd-tabs-link-padding-x;
+    position: relative;
+    padding: $fd-tabs-link-padding-y $fd-tabs-link-padding-x;
+
+    &--error {
+      @include tabs-semantic(var(--sapErrorColor));
+    }
+
+    &--warning {
+      @include tabs-semantic(var(--sapWarningColor));
+    }
+
+    &--success {
+      @include tabs-semantic(var(--sapSuccessColor));
+    }
+
+    &--information {
+      @include tabs-semantic(var(--sapInformationColor));
+    }
+
+    &--neutral {
+      @include tabs-semantic(var(--sapNeutralColor));
+    }
+  }
+
+  &__icon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: $fd-tabs-icon-selection-height;
```

Comment:
should have min-width of 2rem

Reply:
Selectiion size of icon 
Cozy mode - 2.625rem height/width + (2 * 0.625rem) from borders
Compact mode - 1.875rem height/width + (2 * 0.625rem) from borders
General size matches fiori3, Also checked with https://sapui5.hana.ondemand.com/#/entity/sap.m.IconTabBar